webapp-django/crashstats/authentication/management/commands/makesuperuser.py

```diff
@@ -10,7 +10,8 @@ def get_input(text):
 
 class Command(BaseCommand):
 
-    help = 'emailaddress [, otheremailaddress, ...]'
+    def add_arguments(self, parser):
+        parser.add_argument('emailaddress', nargs='+', type=str)
```

Comment:
So you require the argument to be passed with the command, but then it still asks you to write it a second time.